pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -1026,6 +946,7 @@ func convertGateway(r *KubernetesResources) ([]config.Config, map[RouteKey][]str
 				Servers: servers,
 			},
 		}
+		// TODO
```

Comment:
remove or add detail?

Reply:
Meant to remove this entire block, thanks!